bot/cogs/moderation/modlog.py

```diff
@@ -81,6 +83,15 @@ def ignore(self, event: Event, *items: int) -> None:
             if item not in self._ignored[event]:
                 self._ignored[event].append(item)
 
+    @contextmanager
+    def ignore_all(self) -> None:
+        """Ignore all events while inside this context scope."""
+        self._ignore_all = True
+        try:
+            yield
+        finally:
+            self._ignore_all = False
```

Comment:
This won't work so well if the decorator is being used more than once simultaneously because they all share the same variable and one decorator could exit earlier than another.